msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -774,7 +779,7 @@ The following table provides the error codes:
 ## Get real-time Teams meeting events API
 
 > [!NOTE]
-> Real-time Teams meeting events are only supported for scheduled meetings.
+> Real-time Teams meeting events are supported for scheduled and channel meetings.
 
 The user can receive real-time meeting events. As soon as any app is associated with a meeting, the actual meeting start and end time are shared with the bot. The actual start and end time of a meeting are different from scheduled start and end time. The Meeting Details API provides the scheduled start and end time. The event provides the actual start and end time.
 
```

Comment:
I think this __**"You must be familiar with the TurnContext object available through the Bot SDK. The Activity object in TurnContext contains the payload with the actual start and end time. Real-time meeting events require a registered bot ID from the Teams platform. The bot can automatically receive meeting start or end event by adding ChannelMeeting.ReadBasic.Group in the manifest."_**_ is  not very clear. 
Might be clear if we just say:
"If ChannelMeeting.ReadBasic.Group and/or OnlineMeeting.ReadBasic.Chat" are added in the manifest, the bot will automatically start receiving the meeting start/end events for the corresponding meeting types (scheduled and/or channel).

Reply:
Hi @rm3f,

Thank you for your suggestion. however, the content was already part of the article. We've updated the paragraph as per your suggestion.

Thanks,
Yash